gulpfile.js

```diff
@@ -353,151 +353,151 @@ function deployCesium(bucketName, uploadDirectory, cacheControl, done) {
     var errors = [];
 
     return getCredentials()
-    .then(function() {
```

Comment:
There's a lot of changes to the formatting here. Can you reset the file so the only changes are to remove the intended lines?